dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -12,4 +12,50 @@ def index
   def show
     render json: @section.summarize
   end
+
+  # POST /api/v1/sections
+  # Create a new section
+  def create
+    # Validate login_type
+    # TODO: Push validation into model?
+    login_type = (['none', '', nil].include? params[:login_type]) ? 'email' : params[:login_type]
+    return head :bad_request unless %w(word picture email).include? login_type
+
+    valid_script = params[:script] && valid_script_id?(params[:script][:id])
+    script_to_assign = valid_script && Script.find(params[:script][:id])
+
+    section = Section.create!(
```

Comment:
No need. [create](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/persistence.rb#L33-L34) is basically short for `new` and `save`. The only difference (in general) between the bang methods and the non-bang ones is the `!` causes it to raise an error on failure instead of returning boolean.